components/automate-deployment/pkg/target/local_target.go

```diff
@@ -341,7 +340,7 @@ func (t *LocalTarget) SetUserToml(name, config string) error {
 		}
 	}()
 
-	_, err = file.Write([]byte(config))
+	_, err = file.WriteString(config)
```

Comment:
Note to self: No AtomicWriter here?

Reply:
That would be the best but, this will only happen if someone runs the installation twice at the same time in the same machine, right?

Reply:
The fileutils.AtomicWriter also protects against partial write to a file in the case of a crash/powerloss/etc.